torchao/kernel/intmm.py

```diff
@@ -69,7 +69,10 @@ def safe_int_mm(input: torch.Tensor, mat2: torch.Tensor) -> torch.Tensor:
             input = (
                 input.contiguous()
             )  # (it seems the transpose makes cublas check the above j constraint on i)
-        return out_dtype(torch.ops.aten.mm.default, torch.int32, input, mat2)
+        try:
```

Comment:
maybe adding a comment to this would be helpful, how these two branches are handled?

Reply:
The except is executed if it's a float8 dtype on H100, as there's no implementation for addmm_cuda for float8 dtypes